packages/flutter/lib/src/material/navigation_drawer.dart

```diff
@@ -124,6 +125,11 @@ class NavigationDrawer extends StatelessWidget {
   /// Upon updating [selectedIndex], the [NavigationDrawer] will be rebuilt.
   final ValueChanged<int>? onDestinationSelected;
 
+  /// Defines the padding for [NavigationDrawerDestination] widgets (Drawer items).
+  ///
+  /// Defaults to `EdgeInsets.symmetric(horizontal: 12.0)`.
+  final EdgeInsetsGeometry padding;
```

Comment:
`padding` sounds like padding to the drawer widget, I prefer name it to tilePadding or itemPadding,

Reply:
Also, you probably want to add it to navigation_drawer_theme too.